<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AIV_JDZClass</class>
 <widget class="QMainWindow" name="AIV_JDZClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1287</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AIV_JDZ</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>13</y>
      <width>390</width>
      <height>250</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>模式类别</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Gpg_Pattern_number">
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>图片尺寸</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Gpg_Image_rows">
         <property name="text">
          <string>1140</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Gpg_Image_cols">
         <property name="text">
          <string>912</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>码起始点</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Gpg_Pattern_lstrow">
         <property name="text">
          <string>180</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Gpg_Pattern_lstcol">
         <property name="text">
          <string>72</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>格雷码尺寸</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Gpg_Pattern_rows">
         <property name="text">
          <string>768</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Gpg_Pattern_cols">
         <property name="text">
          <string>768</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>码占像素</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Gpg_Code_lines">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Gpg_with_dark">
         <property name="text">
          <string>with_mark</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>标志区域起始行列数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Gpg_Mark_lstrow">
         <property name="text">
          <string>180</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Gpg_Mark_lstcol">
         <property name="text">
          <string>776</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>标志区域行列数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Gpg_Mark_rows">
         <property name="text">
          <string>202</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Gpg_Mark_cols">
         <property name="text">
          <string>64</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="Gpg_generateBtn">
       <property name="text">
        <string>生成格雷码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Gpg_saveBtn">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>21</y>
      <width>378</width>
      <height>222</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>图片尺寸</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Cbg_Image_rows">
         <property name="text">
          <string>1140</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Cbg_Image_cols">
         <property name="text">
          <string>912</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>棋盘格起始行列</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Cbg_Chess_lstrow">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Cbg_Chess_lstcol">
         <property name="text">
          <string>300</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>格子边长</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Cbg_Edge_rows">
         <property name="text">
          <string>50</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Cbg_Edge_cols">
         <property name="text">
          <string>50</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="Qlabel_2">
         <property name="text">
          <string>格子行列数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Cbg_Corner_rows">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Cbg_Corner_cols">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="Cbg_Mark_lstcol">
         <property name="text">
          <string>标志区域起始行列</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Cbg_Mark_lstrow">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="Qlabel">
         <property name="text">
          <string>标志区域的行列数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Cbg_Mark_rows">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Cbg_Mark_cols">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="Cbg_generateBtn">
       <property name="text">
        <string>生成棋盘格</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cbg_saveBtn">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>384</y>
      <width>390</width>
      <height>300</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>图片张数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Ppg_Image_number">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>图片尺寸</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Ppg_Image_rows">
         <property name="text">
          <string>1140</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Ppg_Image_cols">
         <property name="text">
          <string>912</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>相位码起始行列数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Ppg_Pattern_lstrow">
         <property name="text">
          <string>180</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Ppg_Pattern_lstcol">
         <property name="text">
          <string>72</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="Qlabel_3">
         <property name="text">
          <string>相位码区域行列数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Ppg_Pattern_rows">
         <property name="text">
          <string>768</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Ppg_Pattern_cols">
         <property name="text">
          <string>768</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>一个周期相位码行列数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Ppg_peiod_lines">
         <property name="text">
          <string>48</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>标志区域起始行列数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Ppg_Mark_lstrow">
         <property name="text">
          <string>180</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Ppg_Mark_lstcol">
         <property name="text">
          <string>776</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>标志区域行列数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Ppg_Mark_rows">
         <property name="text">
          <string>202</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Ppg_Mark_cols">
         <property name="text">
          <string>64</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_21">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>初始相位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PPg_OriginalPhaseShift">
         <property name="text">
          <string>-3.141592654</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QRadioButton" name="Ppg_with_mark">
       <property name="text">
        <string>with_mark</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Ppg_generateBtn">
       <property name="text">
        <string>生成相位图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Ppg_saveBtn">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1287</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AIV_JDZ.qrc"/>
 </resources>
 <connections/>
</ui>
